chromium_src/chrome/browser/component_updater/chrome_component_updater_configurator.cc

```diff
@@ -15,7 +18,10 @@ namespace component_updater {
 scoped_refptr<update_client::Configurator>
 MakeChromeComponentUpdaterConfigurator(const base::CommandLine* cmdline,
                                        PrefService* pref_service) {
-  return base::MakeRefCounted<BraveConfigurator>(cmdline, pref_service);
+  return base::MakeRefCounted<BraveConfigurator>(
+      cmdline, pref_service,
+      g_browser_process->system_network_context_manager()
```

Comment:
this change was to break the circular dependency for `chrome/browser/net/system_network_context_manager.h`